modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java

```diff
@@ -1098,6 +1102,28 @@ private boolean _equals(WebUrl webUrl1, WebUrl webUrl2) {
 		return true;
 	}
 
+	private long[] _getAccountEntryContactAddressListTypeIds(
```

Comment:
because I don't like this can we just create the contact addresses with the same type?